src/dsp_tools/utils/xmlupload/xmlupload.py

```diff
@@ -361,11 +367,16 @@ def _create_resource(
         )
         logger.warning(log_msg, exc_info=True)
         return None
+    except Exception as err:  # pylint: disable=broad-except
+        msg = f"Unable to create resource '{resource.label}' ({resource.id})"
+        print(f"WARNING: {msg}: {err}")
+        logger.exception(msg)
```

Comment:
In the logs, we should definitively retain the exact stack trace:
```suggestion
        logger.exception(msg, exc_info=True)
```

Reply:
for `logger.exception`, exec_info is True by default

Reply:
Thanks for the hint, I forgot that :-)